src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -517,7 +517,99 @@
+                                         FontFamily="{StaticResource ArtifaktElementRegular}"
+                                         FontSize="12px"
+                                         Foreground="#DFDFDF"
+                                         MaxLength="{TemplateBinding MaxLength}"
+                                         Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Text, UpdateSourceTrigger=PropertyChanged}"
+                                         Tag="{TemplateBinding Tag}"
+                                         TextWrapping="Wrap" />
+                        <TextBlock x:Name="WatermarkLabel"
+                                           Padding="13,13,10,8"
+                                           FontFamily="{StaticResource ArtifaktElementRegular}"
+                                           FontSize="12px"
+                                           Foreground="#DFDFDF"
+                                           IsHitTestVisible="False"
+                                           Opacity="0.5"
+                                           Text="{TemplateBinding Tag}">
+                            <TextBlock.Style>
+                                <Style TargetType="{x:Type TextBlock}">
+                                    <Setter Property="Visibility" Value="Collapsed" />
+                                    <Style.Triggers>
+                                        <DataTrigger Binding="{Binding Text, ElementName=InputTextBox}" Value="">
+                                            <Setter Property="Visibility" Value="Visible" />
+                                        </DataTrigger>
+                                    </Style.Triggers>
+                                </Style>
+                            </TextBlock.Style>
+                        </TextBlock>
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+        <Style.Triggers>
+            <Trigger Property="IsMouseOver" Value="False">
+                <Setter Property="BorderBrush" Value="{StaticResource DarkMidGreyBrush}" />
+            </Trigger>
+        </Style.Triggers>
+    </Style>
+
+    <Style x:Key="LabelStyle" TargetType="TextBlock">
+        <Setter Property="FontFamily" Value="{StaticResource ArtifaktElementBold}" />
+        <Setter Property="FontWeight" Value="Bold" />
+        <Setter Property="FontSize" Value="14px" />
+        <Setter Property="Foreground" Value="#DFDFDF" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="Margin" Value="0,0,0,5" />
+    </Style>
+
+    <Style x:Key="SubLabelStyle" TargetType="TextBlock">
+        <Setter Property="FontFamily" Value="{StaticResource ArtifaktElementRegular}" />
+        <Setter Property="FontSize" Value="12px" />
+        <Setter Property="Foreground" Value="#DFDFDF" />
+        <Setter Property="Margin" Value="0,0,0,5" />
+    </Style>
```

Comment:
Wanted to centralise a bunch of these styles

Reply:
Feel free to! Appreciate that!